### DISCUSSION TITLE
Modern String Stuff (Text blocks, Templates)

### DISCUSSION THREAD

**Discussion by cayhorstmann** [java/devjava-content#23] [New Content Ideas]
### Content Details

An article on what is new with strings: text blocks, templates, and miscellaneous API updates. 

### Website Section

Learn -> Tutorials -> Staying Aware of New Features?

**Comment by carimura** [comment]
This is definitely needed but maybe we wait a bit until record patterns finalize, string templates settle down, and possibly a bit further with pattern matching? The other two are also great but if you want to start here let's make sure to stick to finalized features or make it very clear when something is preview.
